packages/theme/src/default-theme/foundational-styles/colors.ts

```diff
@@ -4,6 +4,7 @@ import fills from './fills'
 
 const colors = {
   background: {
+    default: 'white',
```

Comment:
many components (alert, dropdown) use matching `elevation` theme field, which in turn uses `background.white` as its color source - this way, we should either use the same as the label background or change the rest to use `background.default`

Reply:
Topic for another PR